packages/frontend/src/components/NewsDrawer/NewsItem.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useMemo } from 'react'
+import { BiSolidRocket } from 'react-icons/bi'
+import { Box, Image, Text } from '@chakra-ui/react'
+import { Badge, BadgeLeftIcon } from '@opengovsg/design-system-react'
+import MarkdownRenderer from 'components/MarkdownRenderer'
+import { NEWS_DRAWER_COMPONENT } from 'components/MarkdownRenderer/Components/NewsDrawerComponent'
+import { format } from 'date-fns'
```

Comment:
we dont have date-fns installed. the date library used is luxon. it's dangerous to use a sub-dependency of another library cos u might accidentally remove it without knowing it affects this.